File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    print("Height: ", end='')
    h = get_int('')
    if h >= 1 and h <= 8:
        break
for i in range(h):
    print(" "*(h-i-1), end='')
    print("#"*(i+1), end='')
    print("  ", end='')
    print("#"*(i+1))

File: world/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int n = 0;
    while (n <= 0 || n >= 9)
    {
        n = get_int("Height: ");
    }
    for (int i = 0; i < n; i++)
    {
        for (int j = 0; j < n - i - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
